Sources/PluginLibrary/swift-options.pb.swift

```diff
@@ -10,21 +10,21 @@ import Foundation
 import SwiftProtobuf
 
 
-let Google_Protobuf_FileOptions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
+let Extensions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
```

Comment:
The swiftFieldName on these is a little odd; I'm not really sure what we want.  The top level ones did end up changing with how I've coded this.  I guess the main use of it will be for mirror, in which case, maybe both forms should make it more clear this is an extension, maybe use some notation more like TextFormat does for extensions?  Right now this looks a bit like a normal field which seems wrong.

I also thought extensions weren't supported in json, so I'm not sure why that's needed on these.